internal/v1/server_test.go

```diff
@@ -321,6 +321,88 @@ func TestGetComposeStatus404(t *testing.T) {
 	require.Contains(t, body, "404 during tests")
 }
 
+func TestGetComposeMetadata(t *testing.T) {
+	// simulate osbuild-composer API
+	testPackages := []PackageMetadata{
```

Comment:
This should be `cloudapi.PackageMetadata`. I know the metadata image-builder returns and composer returns are the same, but since this is simulating the composer-api it should use the objects generated from the composer-api spec.